packages/react-sdk/src/components/SlideOverviewBar/SlideOverviewBar.test.tsx

```diff
@@ -414,6 +414,7 @@ async function waitForAnnouncement(
   await waitFor(() => {
     expect(
       // We are not able to access an aria-live region via the testing library
+      // eslint-disable-next-line testing-library/no-node-access
```

Comment:
`aria-live` is only possible to get in testing-library by using external dependencies. This works for us currently, so suppressing the false positive error.